app/furniture/journal/breadcrumbs.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# @see https://github.com/kzkn/gretel
+
+begin
+  crumb :journal_entry do |entry|
+    parent :room, entry.room
+    link entry.headline, [entry.space, entry.room, entry]
+  end
+
+  crumb :new_journal_entry do |entry|
+    parent :room, entry.room
+    link 'Add a Journal Entry', [:new, entry.space, entry.room]
+  end
+
+  crumb :edit_journal_entry do |entry|
+    parent :journal_entry, entry
+    link 'Edit', [:edit, entry.space, entry.room, entry]
+  end
+rescue NoMethodError => e
+  throw e unless e.message.start_with?("undefined method `crumb'")
```

Comment:
What's this about? In which cases would we be throwing an `undefined method crumb` error, and why is that ok to swallow?

Reply:
OK this is HORRIBLE and I *HATE IT* but apparently Gretel's way of re-loading  breadcrumbs uses an `instance_eval`; but if gets autoloaded by Rails and then it explodes.

Reply:
I fixed it by adding breadcrumbs to Zeitwerks ignore directive!